package/Microsoft.Azure.Functions.PowerShellWorker.Package.csproj

```diff
@@ -3,17 +3,15 @@ Copyright (c) Microsoft. All rights reserved.
 Licensed under the MIT license. See LICENSE file in the project root for full license information.
 -->
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Microsoft.Azure.Functions.PowerShellWorker.Common.props))\Microsoft.Azure.Functions.PowerShellWorker.Common.props" />
```

Comment:
Why not use `<Import Project="..\Microsoft.Azure.Functions.PowerShellWorker.Common.props" />`?

Reply:
I did this so you would be able to build the project from anywhere on the file system, but it's probably not needed since build.ps1 is our entry point.

Reply:
Fixed